### PR TITLE
docs: add new android tutorial for web searching automation

### PR DESCRIPTION
**Pull request by HaramChoi-askui**
Added a new tutorial for web searching in android devices.

### CODE REVIEW

File: docs/docs/general/06-Tutorials/android-search-in-browser.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+  });
+});
+```
+
+and run,
+```bash
+npx jest test/my-first-askui-test-suite.test.ts --config ./test/jest.config.ts
+```
+
+![annotated-chrome](images/annotation-chrome.png)
```
Comment (thinh-askui) [inline docs/docs/general/06-Tutorials/android-search-in-browser.md:138]: Did you use the `annotation.png`? If not you can just delete it

**Review by thinh-askui — CHANGES_REQUESTED** [review]
Good job! And thank you for helping with the setup :)
I have some remarks but this shouldn't be big changes

**Comment by HaramChoi-askui** [comment]
@thinh-askui Thank you for the review. I have applied the points you mentioned. The unused .png has beed deleted. Thanks for that!

**Review by thinh-askui — APPROVED** [review]
Good Job Haram!

**Pull request merged**